### PR TITLE
AYS-400 | roleName filter error fix

### PR DESCRIPTION
**Pull request by gafur-demirci**
### Checklist

Before submitting your pull request, ensure the following:

- [ ] **Change Type:**
- [x] Bug fix
- [ ] Feature addition
- [ ] Documentation update
- [ ] Performance improvement
- [ ] Other (please specify)

- [x] **Summary:**

  - roleName input validation added and min two characters can filter dataTable functionality added.

- [x] **Title and Branch Naming Conventions:**

  - The pull request title follows the
    standard: [Pull Request Naming Conventions](CONTRIBUTING.md#pull-request-naming-conventions).
  - The branch name follows one of the conventions outlined in
    the [Branch Naming Conventions](CONTRIBUTING.md#branch-naming-conventions).

- [x] **Local Testing:**

  - I have tested my changes locally on my browser, and they are working as expected.

- [x] **Code Quality:**

  - The code is formatted according to the project's coding guidelines and style.
  - The code has been reviewed to ensure its quality.
  - The code does not contain any issues flagged by ESLint.

- [ ] **Documentation:**

  - Necessary documentation has been added or existing documentation has been updated.

- [x] **Reviewers and Assignees:**

  - Default reviewers have been assigned to this pull request.
  - Assignees have been added if necessary.

  - [x] **Related Issue:**
  - [AYS-400](https://afetyonetimsistemi.atlassian.net/browse/AYS-400)

- [x] **Labels and Associations:**
  - No specific actions are required in the Labels and Associations section for this pull request.

[AYS-400]: https://afetyonetimsistemi.atlassian.net/browse/AYS-400?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sanshigo345 — CHANGES_REQUESTED** [review]
tek karakter girişi ile hala backend'e istek atıyor

File: src/i18n/locales/en.json [hunk 1]
```diff
@@ -85,6 +85,7 @@
   "emailAddress": "E-mail Address",
   "seatingCountValidationMessage": "Only positive numbers with a maximum of {{field}} digits are allowed.",
   "referenceNumberValidationMessage": "Only positive numerical values are allowed.",
+  "nameValidationMessage": "Only letters values are allowed.",
   "adminRegistrationApplications": {
```
Comment (sanshigo345) [inline src/i18n/locales/en.json:89]: "Only letters values are allowed." cümle yapısı olarak yanlış bir de 

> [Minimum 2, maksimum 255 karakter uzunluğunda ve özel karakter içermeyen bir değer alabilir.](https://afetyonetimsistemi.atlassian.net/wiki/spaces/AYS/pages/32636949/Y+netim+Paneli+F1+-+Rol+Y+netimi+Analizi#Rol-Olu%C5%9Fturma)

Comment (gafur-demirci) [reply src/i18n/locales/en.json]: Hata mesajı içeriği güncellendi.

Comment (sanshigo345) [reply src/i18n/locales/en.json]: "Role name should have minimum 2, maximum 255 characters. Special characters not allowed."

daha kısa ve daha açıklayıcı bence. bu arada linkteki yeni bilgiler ışığında validasyon değiştirilmesi gerekebilir.

Comment (gafur-demirci) [reply src/i18n/locales/en.json]: Hata mesajı daha kısa ve anlaşılır olacak şekilde düzenlendi.

Comment (sanshigo345) [reply src/i18n/locales/en.json]: TR translationı da bu değişime göre güncellemek lazım, aklında bulunsun

Comment (gafur-demirci) [reply src/i18n/locales/en.json]: TR kısmı da tekrar güncellendi.

**Review by sanshigo345 — APPROVED** [review]

**Pull request merged**